### DISCUSSION TITLE
env meta tags are not generating when deploy custom client

### DISCUSSION THREAD

**Discussion by baechoofarm** [mozilla/hubs#5174] [Hubs Cloud Questions]
### **meta tags before deploy**
![image](https://user-images.githubusercontent.com/9635450/155179078-c64ab0cf-69f7-49dd-ad64-76fc8a02123a.png)

### **after deploy**
![image](https://user-images.githubusercontent.com/9635450/155179260-729335fa-3e24-4955-bf29-2d9ac01427af.png)

When I deploy my custom client (based on aws), no env meta tags are generated.
So... my admin pages don't work after deploy. (if I undeploy, it works)

How can I do?

**Comment by matthewbcool** [comment]
Have you tried adding this tags via the admin panel? Check out the app settings/advanced settings there and let us know if that solves.
